docs/en/writing-running-appium/other/reset-strategies.md

```diff
@@ -2,5 +2,7 @@
 
 | Automation Name | default | fullReset | noReset |
 | --------------- | ------- | --------- | ------- |
-| iOS (including XCUITest) | Shut down sim after test. Do not destroy sim. Do not uninstall app from real device. | Uninstall app after real device test, destroy Simulator after sim test | Do not destroy or shut down sim after test. Start tests running on whichever sim is running, or device is plugged in |
+| iOS (including XCUITest) | Shut down sim after test. Do not destroy sim. Uninstall app from Simulator(1). Do not uninstall app from real device. | Uninstall app before and after real device test, destroy Simulator before and after sim test. They happen only _before_ if `resetOnSessionStartOnly: true` is provided | Do not destroy or shut down sim after test. Start tests running on whichever sim is running, or device is plugged in |
```

Comment:
I prefer `Uninstall app from Simulator*` and then below ... *: Make sure the local...

Also, I would change the whole thing to,  

```
If it's a simulator, uninstalls the app-under-test* but does not destroy simulator. If it's a real device, does not uninstall app-under-test.
```